testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/observability/opentelemetry/exporter/InMemorySpanExporter.java

```diff
@@ -39,8 +67,11 @@ public List<SpanData> getFinishedSpanItems(int spanCount) {
 
     public void assertSpanCount(int spanCount) {
         try {
-            Awaitility.await().pollDelay(3, SECONDS).atMost(30, SECONDS)
+            System.err.println("***** Waiting " + pollDelay + " for spans to be ready");
+            Awaitility.await().pollDelay(pollDelay)
+                    .atMost(pollDelay.multipliedBy(2))
                     .untilAsserted(() -> Assert.assertEquals(spanCount, finishedSpanItems.size()));
+            System.err.println("***** All spans found");
```

Comment:
Please either comment out all these System.err calls or use a logger (which will put them in target/test.log).

Generally, commenting out is better as logging adds CI burden and it adds up.

Reply:
That wasn't supposed to make it to GH. :) I'll clean that up.